### ISSUE TITLE
Audio maybe?

### ISSUE THREAD

**Issue by cookiboy** [circleguard/circleguard#170]
i think adding audio would be a good idea and generally make it better, i find it kind of hard to analyze the ur of a suspected relax without audio so if it is possible, i think it would be a good (major) upgrade to circleguard.

**Comment by tybug** [comment]
yeah I'd like to see this happen eventually, will likely require us to pull from an existing osu installation which will take some work

**Comment by cookiboy** [comment]
nice to see that you think it would be a good idea as well!
